File: Components/BlogML/Pair.vb
Imports System.Collections.Generic
Imports System.Text


Namespace BlogML

 ''' <summary>
 ''' A serializable keyvalue pair class
 ''' </summary>
 Public Structure Pair(Of K, V)
  Public Key As K
  Public Value As V
  Public Sub New(key As K, value As V)
   Me.Key = key
   Me.Value = value
  End Sub
 End Structure

End Namespace

File: Server/Core/BlogML/Pair.cs

namespace DotNetNuke.Modules.Blog.BlogML
{

  /// <summary>
 /// A serializable keyvalue pair class
 /// </summary>
  public struct Pair<K, V>
  {
    public K Key;
    public V Value;
    public Pair(K key, V value)
    {
      Key = key;
      Value = value;
    }
  }

}